src/components/clientDashboard/enrollments/ClientAssessments.tsx

```diff
@@ -25,16 +25,16 @@ type AssessmentType = NonNullable<
 >['nodes'][0];
 
 const columns: ColumnDef<AssessmentType>[] = [
+  {
+    header: 'Assessment Type',
+    render: (assessment) => formRoleDisplay(assessment),
+  },
   {
     header: 'Assessment Date',
     render: (a) => <AssessmentDateWithStatusIndicator assessment={a} />,
     linkTreatment: true,
```

Comment:
Oh can you move this `linkTreatment` over to the assessment type column? I think that is what we agreed on. It makes sense to be visually clickable because it provides more context as to what the item is.